Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.6
+// swift-tools-version:5.7
```

Comment:
Should we increase to 5.9?

Reply:
Our [support policy](https://github.com/amzn/smoke-aws/blob/main/docs/Support_Policy.md) states that we have a trailing window of supporting the two most recent compiler versions and we can drop support for the third most recent compiler version six months after the release of the latest compiler version. So according to this we should support 5.7 until the 18th of March 2024.